### PR TITLE
githubteacher's tampa places to eat

### PR DESCRIPTION
**Pull request by githubteacher**
@hollenberry any suggestions for edits?

### CODE REVIEW

**Comment by carolynshin** [comment]
@githubteacher, have you tried [Sulphur Springs Sandwich Shop](https://www.yelp.com/biz/sulphur-springs-sandwich-shop-tampa)?

File: githubteacher-tampa.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Places to Eat
```
Comment (carolynshin) [inline githubteacher-tampa.md:1]: @beardofedu: Are there good asian restaurants in Tampa?

File: githubteacher-tampa.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+ - Dunedin Brewery :beer:
+ - Demitri's on the Water
```
Comment (altafkassam) [inline githubteacher-tampa.md:3]: Can you dine outside? How's the :beer: selection? Do they have a happy hour?

**Review by hollenberry — CHANGES_REQUESTED** [review]
Please add 🍵 shops.

File: githubteacher-tampa.md (hunk 1 above)
Comment (carolynshin) [reply githubteacher-tampa.md:1]: @beardofedu  THAI FOOD

Comment (githubteacher) [reply githubteacher-tampa.md:1]: any specific asian food you are looking for? @carolynshin  🍣 🇹🇭 🇰🇷

File: githubteacher-tampa.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+   - Eat outside, feed the fish / :duck:
+ - Thai Restaurant on Sunset Blvd (Clearwater)
```
Comment (carolynshin) [inline githubteacher-tampa.md:5]: This is not helpful. I hope in your next iteration you provide the actual restaurant name.

**Review by hollenberry — APPROVED** [review]

**Comment by hollenberry** [comment]
👍

**Pull request merged**